_data/talks.yml

```diff
@@ -374,3 +374,74 @@
   date: 3rd Oct
   time: 20:00 pm
   type: special
+
+# Schedule for 4th Oct
+
+- title: Getting Started with Python
+  track: 1
+  date: 4th Oct
+  time: 11:30 am
+  type: workshop
+  speaker: Evan Smith
+  hyperlink: https://in.pycon.org/cfp/2020/proposals/getting-started-with-python~b4x8V/
+
+- title: Python for Computational Social Science with Text, Networks and clever data games
+  track: 2
+  date: 4th Oct
+  time: 11:30 am
+  type: workshop
+  speaker: Bhargav Srinivasa
+  hyperlink: https://in.pycon.org/cfp/2020/proposals/python-for-computational-social-science-with-text-networks-and-clever-data-games~dyPPE/
+
+- title: 
+  track: 3
+  date: 4th Oct
+  time: 11:30 am
+  type: workshop
+  speaker: 
+  hyperlink: 
+
+- title: 
+  track: 4
+  date: 4th Oct
+  time: 11:30 am
+  type: workshop
+  speaker: 
+  hyperlink: 
+
+- title: Break / Networking
+  track: all
+  date: 3rd Oct
+  time: 14:00 pm
```

Comment:
@anistark If we are following 24 hours format, then am/pm not required

Reply:
Yeah. That's how the other talks are written right now 😅 Also need to add `IST`.

Reply:
Hmm, we can remove from the other places as well

Reply:
@sayanchowdhury should we remove am/pm ?

Reply:
If they are 24 hour please remove them as well. Also, since the current schedule doesn't have IST, let's not put it here. It is a separate concern.

Reply:
removed.